### PR TITLE
Some corrections for the documentation

### PR DESCRIPTION
**Pull request by KieranDelamotte**
This PR contains some corrections for the documentation.

It solves the issue #1631

### CODE REVIEW

**Review by jschueller — APPROVED** [review]
You can retarget your pr to the 1.16 branch if you want this to land in 1.16,
otherwise +1

**Comment by KieranDelamotte** [comment]
@jschueller 
Thank for your input. How can I retarget ? Also when do you stop accepting PRs for 1.16 ?

**Comment by jschueller** [comment]
I think you have to reopen it to choose the branch. We plan to release 1.16 in 3 weeks.

File: python/doc/examples/data_analysis/estimate_dependency_and_copulas/plot_independence_test.py [hunk 1]
```diff
@@ -11,28 +11,25 @@
...
 #
 # .. math::
-#      D = \displaystyle 1-\frac{6\sum_{i=1}^n (r_i - s_i)^2}{n(n^2-1)}
-#
-#   where :math:`r_i = rank(x_i)` and  :math:`s_i = rank(y_i)`. :math:`D` is such that :math:`\sqrt{n-1}D` tends towards the gaussian (0,1) distribution.
-#
-#
+#      D = 1-\frac{6\sum_{i=1}^n (r_i - s_i)^2}{n(n^2-1)}
 #
+# where :math:`r_i = rank(x_i)` and  :math:`s_i = rank(y_i)`. :math:`D` is such that :math:`\sqrt{n-1}D` tends towards the gaussian (0,1) distribution.
```
Comment (josephmure) [inline python/doc/examples/data_analysis/estimate_dependency_and_copulas/plot_independence_test.py:32]:
```suggestion
# where :math:`r_i = rank(x_i)` and  :math:`s_i = rank(y_i)`. :math:`D` is such that :math:`\sqrt{n-1}D` tends towards the standard normal distribution.
```
We are trying to have a more consistent terminology: normal distributions and Gaussian vectors/processes.

File: python/doc/examples/probabilistic_modeling/distributions/plot_createYourOwnDist-vMB.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # %%
 # We want to create a distribution which cumulated distribution (cdf) is denoted by :math:`F` when the quantile function :math:`F^{-1}` is known. In order to implement this, we use the `CompositeDistribution <http://openturns.github.io/openturns/latest/user_manual/_generated/openturns.CompositeDistribution.html#openturns.CompositeDistribution>`_ class.
```
Comment (josephmure) [inline python/doc/examples/probabilistic_modeling/distributions/plot_createYourOwnDist-vMB.py:8]:
```suggestion
# We want to create a distribution with CDF :math:`F` from the quantile function :math:`F^{-1}`. In order to implement this, we use the `CompositeDistribution <http://openturns.github.io/openturns/latest/user_manual/_generated/openturns.CompositeDistribution.html#openturns.CompositeDistribution>`_ class.
```
This formulation is simpler in my opinion. What do you think?

File: python/doc/examples/probabilistic_modeling/distributions/plot_createYourOwnDist-vMB.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
...
 
-#
+# %%
 # We know that the random variable :math:`X` is distributed according to :math:`F` if and only if :math:`U=F(X)` is distributed according to the uniform distribution in the :math:`[0,1]` interval, i.e. :math:`U=F(X) \sim \mathcal{U}(0,1)`. Hence, if :math:`U \sim  \mathcal{U}(0,1)` then :math:`X=F^{-1}(U)` is distributed according to :math:`F`.
 #
 # In this example, we want to create a distribution which cdf :math:`F: \mathbb{R} \rightarrow [0,1] ` writes, with :math:`\rho > 1`:
```
Comment (josephmure) [inline python/doc/examples/probabilistic_modeling/distributions/plot_createYourOwnDist-vMB.py:13]:
```suggestion
# In this example, we want to create a distribution with CDF :math:`F: \mathbb{R} \rightarrow [0,1] `parametrized by :math:`\rho > 1`:
```

**Review by josephmure — COMMENTED** [review]
A few suggestions. Not sure about all of them.

**Comment by josephmure** [comment]
Also, you can edit your initial message to replace "It solves the issue #1631" with "Resolves #1631". This way Github will [understand the link](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) between this PR and the issue. Currently it only understands the link between one of the commits and the issue. The advantage is that links between issue and PR will appear in the sidebar and make browsing easier.

**Review by regislebrun — APPROVED** [review]

File: python/src/CMakeLists.txt [hunk 4]
```diff
@@ -1002,6 +1005,19 @@ if (SPHINX_FOUND)
...
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/deflection_tube.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/deflection_tube.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/flood_model.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/flood_model.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/ishigami_function.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/ishigami_function.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/logistic_model.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/logistic_model.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/stressed_beam.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/stressed_beam.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/viscous_free_fall.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/viscous_free_fall.py)
```
Comment (jschueller) [inline python/src/CMakeLists.txt:1020]: we may use the "files (glob)" command to list these, then copy them in a loop

Comment (KieranDelamotte) [reply python/src/CMakeLists.txt]: @jschueller Done ! Thanks for pointing  it out.

File: python/doc/examples/data_analysis/distribution_fitting/plot_estimate_non_parametric_distribution.py [hunk 5]
```diff
@@ -1,62 +1,269 @@
 """
-Non parametric distribution fitting
-===================================
+Fit a non parametric distribution
```
Comment (josephmure) [inline python/doc/examples/data_analysis/distribution_fitting/plot_estimate_non_parametric_distribution.py:2]:
```suggestion
Fit a non-parametric distribution
```

File: python/doc/examples/data_analysis/distribution_fitting/plot_estimate_non_parametric_distribution.py [hunk 6]
```diff
@@ -1,62 +1,269 @@
...
+=================================
 """
 # %%
-# In this example we are going to estimate a non parametric distribution using the kernel smoothing method.
+# In this example we are going to estimate a non parametric distribution using the kernel smoothing method. After a short introductory example we focus on a few basic features of the API :
```
Comment (josephmure) [inline python/doc/examples/data_analysis/distribution_fitting/plot_estimate_non_parametric_distribution.py:6]:
```suggestion
# In this example we are going to estimate a non-parametric distribution using the kernel smoothing method. After a short introductory example we focus on a few basic features of the API :
```

File: python/doc/examples/data_analysis/distribution_fitting/plot_estimate_non_parametric_distribution.py (hunk 5 above)
Comment (josephmure) [inline python/doc/examples/data_analysis/distribution_fitting/plot_estimate_non_parametric_distribution.py:2]:
```suggestion
Fit a non-parametric distribution
```

File: python/doc/examples/data_analysis/distribution_fitting/plot_estimate_non_parametric_distribution.py (hunk 6 above)
Comment (josephmure) [inline python/doc/examples/data_analysis/distribution_fitting/plot_estimate_non_parametric_distribution.py:6]:
```suggestion
# In this example we are going to estimate a non-parametric distribution using the kernel smoothing method. After a short introductory example we focus on a few basic features of the API :
```

**Comment by KieranDelamotte** [comment]
I think it is time to add this PR to the release as it fixes some documentation issues. Is there anything I have to do for this PR to be merged ?

Concerning the cantilever beam example, I  updated the parameters with @mbaudin47 suggestions. As a result the failure probability decreases from 5e-3 to approx 1e-7.

**Comment by jschueller** [comment]
it looks ok, I'll merge when the ci is ok

**Pull request merged**